search-cysticFibrosis.rq

```diff
@@ -0,0 +1,32 @@
+#+ endpoint: http://ejprd.fair-dtls.surf-hosted.nl:8085/blazegraph/namespace/kb/sparql
```

Comment:
Can we generalize this query to search for any given ordo URL?

Reply:
Do you want me to create a new query or modify this one?

Reply:
Can you modify it?